docker-compose.override.example.yml

```diff
@@ -90,6 +90,10 @@ services:
     # image: docker.io/your-id/redisearch:${REDISEARCH_IMAGE_TAG}-0
     ports:
       - ${REDISEARCH_PUBLISH_PORT}:6379
+  redisearch-slave:
```

Comment:
Put a blank line here (between line 92 & 93). Each services should be grouped separately.